packages/shared/src/utils/RequestSizeInput.tsx

```diff
@@ -53,7 +53,10 @@ export const RequestSizeInput: React.FC<RequestSizeInputProps> = ({
   testID,
   unitText,
 }) => {
-  const parsedRequestSizeValue = parseInt(defaultRequestSizeValue, 10);
+  const parsedRequestSizeValue = parseInt(
+    defaultRequestSizeValue as string,
```

Comment:
```suggestion
    String(defaultRequestSizeValue)
```
This would guarantee you string everytime

Reply:
ack, will update...
though passing `number` doesn't affect the functionality...